### PR TITLE
Pallets page. First contribution

### LINKED ISSUE #86: Pallets description

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: src/pallets.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # Pallets
 
-<!-- TODO:
-Move the relevant information from https://soramitsu.atlassian.net/wiki/spaces/SB/pages/3702784176/System+Architecture
+In the context of SORA, pallets play a crucial role in the development of the SORA Network and its associated applications. SORA is a decentralized autonomous economy (DAE) built on the [Substrate](https://substrate.io/) framework, and it utilizes Substrate's [pallets](https://docs.substrate.io/learn/welcome-to-substrate/) to implement its specific functionalities and features.
```
Comment (WRRicht3r) [inline src/pallets.md]:
```suggestion
In the context of SORA, pallets play a crucial role in the development of the network and its associated applications. SORA is a decentralized autonomous economy (DAE) built on the [Substrate](https://substrate.io/) framework, and it utilizes Substrate's [pallets](https://docs.substrate.io/learn/welcome-to-substrate/) to implement its specific functionalities and features.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/pallets.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
 # Pallets
 
-<!-- TODO:
-Move the relevant information from https://soramitsu.atlassian.net/wiki/spaces/SB/pages/3702784176/System+Architecture
+In the context of SORA, pallets play a crucial role in the development of the network and its associated applications. SORA is a decentralized autonomous economy (DAE) built on the [Substrate](https://substrate.io/) framework, and it utilizes Substrate's [pallets](https://docs.substrate.io/learn/welcome-to-substrate/) to implement its specific functionalities and features.
 
--->
+By leveraging Substrate's pallets, SORA can customize its blockchain [runtime](https://docs.substrate.io/learn/runtime-development/) to meet its unique requirements and design principles. The pallets enable SORA to incorporate key functionalities such as token management, decentralized governance, bridging to other networks, liquidity provision, and more.
```
Comment (outoftardis) [inline src/pallets.md]:
```suggestion
By leveraging Substrate's pallets, SORA can customize its blockchain [runtime](https://docs.substrate.io/learn/runtime-development/) to meet its unique requirements and design principles. The pallets enable SORA to incorporate key functionalities such as token management, decentralized governance, bridges to other networks, liquidity provision, and more.
```

File: src/pallets.md [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+
+Developers working on SORA can refer to the Substrate documentation to understand the underlying concepts and usage of pallets. This knowledge will aid them in effectively utilizing the pallets within the SORA ecosystem, customizing them to fit the unique requirements of SORA.
+
+To delve deeper into the connection between pallets and SORA, developers can also refer to the [Rust documentation (rustdoc)](https://docs.rs/) for the SORA pallets. Currently, there are two Rust projects being developed: SORA Mainnet and SORA Parachain. Each project has its own dedicated repo:
+
+- For the SORA Mainnet, you can find the Rust documentation at https://github.com/sora-xor/sora2-network.
+- For the SORA Parachain, the Rust documentation is available at https://github.com/sora-xor/sora2-parachain.
```
Comment (outoftardis) [inline src/pallets.md]:
```suggestion
To delve deeper into the connection between pallets and SORA, developers can also refer to the [Rust documentation (rustdoc)](https://docs.rs/) for the SORA pallets. Currently, there are two Rust projects being developed: [SORA Mainnet](https://github.com/sora-xor/sora2-network) and [SORA Parachain](https://github.com/sora-xor/sora2-parachain).
```

or if you want to preserve the list:
 
```suggestion
To delve deeper into the connection between pallets and SORA, developers can also refer to the [Rust documentation (rustdoc)](https://docs.rs/) for the SORA pallets. Currently, there are two Rust projects being developed: SORA Mainnet and SORA Parachain. Each project has its own dedicated repository:

- [SORA Mainnet](https://github.com/sora-xor/sora2-network)
- [SORA Parachain](https://github.com/sora-xor/sora2-parachain)
```

**Review by outoftardis — APPROVED** [review]

File: src/pallets.md [hunk 4]
```diff
@@ -1,6 +1,13 @@
 # Pallets
 
-<!-- TODO:
-Move the relevant information from https://soramitsu.atlassian.net/wiki/spaces/SB/pages/3702784176/System+Architecture
+In the context of SORA, pallets play a crucial role in the development of the network and its associated applications. SORA is a decentralized autonomous economy (DAE) built on the [Substrate](https://substrate.io/) framework, and it utilizes Substrate's [pallets](https://docs.substrate.io/learn/welcome-to-substrate/) to implement its specific functionalities and features.
 
--->
+By leveraging Substrate's pallets, SORA can customize its blockchain [runtime](https://docs.substrate.io/learn/runtime-development/) to meet its unique requirements and design principles. The pallets enable SORA to incorporate key functionalities such as token management, decentralized governance, bridges to other networks, liquidity provision, and more.
```
Comment (WRRicht3r) [inline src/pallets.md]:
```suggestion
By leveraging Substrate's pallets, SORA can customize its blockchain [runtime](https://docs.substrate.io/learn/runtime-development/) to meet its unique requirements and design principles. These pallets enable SORA to incorporate key functions such as token management, decentralized governance, bridges to other networks, liquidity provision, and more.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**